### PR TITLE
Expand conf

### PR DESCRIPTION
**Pull request by jgw96**
- Add more sessions
- Add refresher
- Add virtualScroll
- Add toast

### CODE REVIEW

**Comment by brandyscarney** [comment]
Added a few commits, only issue I found: the items are overlapping on desktop:

screen shot 2017-04-13 at 1 43 28 pm

**Comment by jgw96** [comment]
Hmmm interesting, looking into this.

File: package.json [hunk 1]
```diff
@@ -33,7 +33,9 @@
     "zone.js": "^0.8.4"
   },
   "devDependencies": {
-    "@ionic/app-scripts": "1.2.4",
+    "@ionic/app-scripts": "1.3.2",
+    "@ionic/cli-plugin-cordova": "0.0.12",
```
Comment (gianpaj) [inline package.json]: what are these dependencies? what do they do/replace/intended for? I can't find anything here https://www.npmjs.com/package/@ionic/cli-plugin-cordova or in the cli repo https://github.com/driftyco/ionic-cli

Comment (brandyscarney) [inline package.json]: @gianpaj They're needed for CLI v3: https://github.com/driftyco/ionic-cli/tree/v3#install

**Pull request merged**